packages/Webkul/Shop/src/Resources/lang/ar/app.php

```diff
@@ -7,7 +7,7 @@
             'email-not-exist'      => 'لا يمكننا العثور على مستخدم بهذا البريد الإلكتروني',
             'email'                => 'البريد الإلكتروني',
             'forgot-password-text' => 'إذا نسيت كلمة المرور الخاصة بك، يمكنك استعادتها عن طريق إدخال عنوان البريد الإلكتروني الخاص بك.',
-            'footer'               => '© حقوق الطبع والنشر 2010 - 2022، Webkul Software (مسجلة في الهند). جميع الحقوق محفوظة.',
+            'footer'               => '© حقوق الطبع والنشر 2010 - :currentYear، Webkul Software (مسجلة في الهند). جميع الحقوق محفوظة.',
```

Comment:
current_year

Reply:
Okay @devansh-webkul , I will change it from currentYear to current_year.